authorization-common/src/main/java/com/sequenceiq/authorization/service/UmsAuthorizationService.java

```diff
@@ -26,35 +26,56 @@
     @Inject
     private GrpcUmsClient umsClient;
 
-    public void checkRightOfUserForResource(String userCrn, AuthorizationResource resource, ResourceAction action, String unauthorizedMessage) {
-        if (!umsClient.checkRight(userCrn, userCrn, RightUtils.getRight(resource, action), getRequestId())) {
+    // ACCOUNT LEVEL
+
+    public void checkRightOfUser(String userCrn, AuthorizationResourceType resourceType, AuthorizationResourceAction action) {
+        String right = RightUtils.getRight(resourceType, action);
+        String unauthorizedMessage = String.format("You have no right to perform %s. This requires one of these roles: %s. "
+                        + "You can request access through IAM service from an administrator.",
```

Comment:
we are giving this error message in every non happy use case and i think we need a logic to find out what role is needed in a case of right, we had a logic for that but it was ugly and it can become obsolete quickly, so I removed it months ago, anyway we need to find a good and maintainable logic for that

Reply:
i have added this to the TODO list